examples/simple/cmd/res/configuration.toml

```diff
@@ -2,7 +2,7 @@
 Host = "localhost"
 Port = 49990
 ConnectRetries = 3
-HealthCheck = ""
+HealthCheck = "/api/v1/ping"
```

Comment:
pls make it in separate commit. This one related to test simple-device service in examples not for device-sdk

Reply:
yes, sure. But 'simple' directory is not related to device-sdk. It even can be in separate repository. So pull request should contain 2 separate commits. One for device-sdk and another onw for simple-device example.

Reply:
Sorry, I got your point. It's need more commit history to describe changes.